### DISCUSSION TITLE
Potential ambiguity in the definition of Flow channel area when combined with num_rods

### DISCUSSION THREAD

**Discussion by Jake-HW-Miles** [idaholab/moose#28263] [Q&A Modules: Thermal Hydraulics]
Hi there,
This may be a bit of a silly questions. However I want to make sure I am not barking up the wrong tree. 
So I am constructing a Heat exchanger using components and am unsure which area to use. If I have a heat exchanger which has N tubes, each with an area of A, should be flow channel area be N*A or simply A? (I have seen examples of both)

**Comment by GiudGiud** [comment]
Hello

I could see it coded either way, but it's likely expecting a single channel area. The parameter should say
which object is it so we can update the parameter description?

Guillaume

**Reply by Jake-HW-Miles** [reply]
I have been using `HeatStructureCylindrical` which is why I mention "num_rods", however I assume that all HS are the same. Thank you for clearing this up!

**Reply by Jake-HW-Miles** [reply]
But the Area (A) is stemming from the required input for `FlowChannel1Phase`

**Reply by GiudGiud** [reply]
So you have a component that is both a FlowChannel1Phase and a HeatstructureCylindrical? That does not seem right?

The volume in HSCylindrical is
```
      _volume.push_back(_num_rods * M_PI * (r[i + 1] * r[i + 1] - r[i] * r[i]) * _length);
```
so the number of rods is multiplying the single-rod design for that class

**Reply by Jake-HW-Miles** [reply]
Apologies for not being clear, this is my heat exchanger group (reduced input):
```
    [hx]
        [pipe]
            type = FlowChannel1Phase
            position = '${pri_x_core} ${pri_y_core} 0.'
            orientation = ${right}
            length = ${hx_length}
            n_elems = ${n_hx_elements}
            A = '${fparse **tube_N** * pi * hx_dia_pri * hx_dia_pri * 0.25}'
            D_h = ${hx_dia_pri}
      
            fp = coolant
            initial_p = ${pri_press}
            initial_vel = ${vel_ini_pri}
            initial_T = ${initial_T_hx}
        []
        [wall]
            type = HeatStructureCylindrical
            orientation = ${right}
            position = '${pri_x_core} ${pri_y_core} 0.'
            length = ${hx_length}

            n_elems = ${n_hx_elements}
            
            names = 'hx_wall'
            widths = '${hx_wall_thickness}'
            n_part_elems = 1
            solid_properties = 'alloy625'
            solid_properties_T_ref = '${units 600 degC -> K}'
        
            initial_T = ${initial_T_hx}
            inner_radius = '${fparse hx_dia_pri / 2}'
            num_rods = ${**tube_N**}
        []
    []
```

I have tried to highlight references to the number of hx tubes, but based on the previous answer, it seems I should remove it from the flow channel

**Reply by GiudGiud** [reply]
I think so.
@joshuahansel to check

**Comment by joshuahansel** [comment]
Good question, documentation is lacking on this topic.

Your flow channel should represent the _total_ flow area. Make sure to set the heated perimeter parameter `P_hf` in your `HeatTransferFromHeatStructure1Phase` component to correspond to the total as well. `D_h` should be the same for individual and total.
